as-vending/main.go

```diff
@@ -70,40 +72,40 @@ func main() {
 
 	var err error
 
-	err = edgexSdk.AddRoute("/boardStatus", vendingState.BoardStatus, "POST")
-	errorAddRouteHandler(edgexSdk, err)
+	err = service.AddRoute("/boardStatus", vendingState.BoardStatus, "POST")
+	errorAddRouteHandler(lc, err)
 
-	err = edgexSdk.AddRoute("/resetDoorLock", vendingState.ResetDoorLock, "POST")
-	errorAddRouteHandler(edgexSdk, err)
+	err = service.AddRoute("/resetDoorLock", vendingState.ResetDoorLock, "POST")
+	errorAddRouteHandler(lc, err)
 
-	err = edgexSdk.AddRoute("/maintenanceMode", vendingState.GetMaintenanceMode, "GET", "OPTIONS")
-	errorAddRouteHandler(edgexSdk, err)
+	err = service.AddRoute("/maintenanceMode", vendingState.GetMaintenanceMode, "GET", "OPTIONS")
+	errorAddRouteHandler(lc, err)
```

Comment:
Wrap in controller

Reply:
I will wrap in controller in next PR as enhancement